syncany-lib/src/it/java/org/syncany/tests/integration/database/DatabaseReconciliatorTest.java

```diff
@@ -704,10 +705,14 @@ private void testFromMachinePerspective(String localMachineName, DatabaseVersion
 		TestResult actualTestResult = new TestResult();
 
 		// Get 'local' branch
-		DatabaseBranch localBranch = allBranches.getBranch(localMachineName);
+		DatabaseBranch localBranch = allBranches.get(localMachineName);
 
 		// Get all the other ones (clone 'all', and remove local)
-		DatabaseBranches unstitchedRemoteBranches = allBranches.clone();
+		Map<String, DatabaseBranch> unstitchedRemoteBranches = new TreeMap<String, DatabaseBranch>(){{
```

Comment:
Double curly braces are considered an anti pattern: https://blog.jooq.org/2014/12/08/dont-be-clever-the-double-curly-braces-anti-pattern/

See my comment on the conversation page. You alreday lost some information. While `allBranches.clone()` is quite readable and understandable, you needed to take a detour here by using the curly braces pattern.